static/app/components/events/viewHierarchy/index.tsx

```diff
@@ -0,0 +1,60 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+
+import {RenderingSystem} from './renderingSystem';
+import {Tree} from './tree';
+
+export type ViewHierarchyWindow = {
+  alpha: number;
+  height: number;
+  id: string;
+  type: string;
+  visible: boolean;
+  width: number;
+  x: number;
+  y: number;
+  children?: ViewHierarchyWindow[];
+  depth?: number;
+  identifier?: string;
+};
+
+export type ViewHierarchyData = {
+  rendering_system: string;
+  windows: ViewHierarchyWindow[];
+};
+
+type ViewHierarchyProps = {
+  viewHierarchy: ViewHierarchyData;
+};
+
+function ViewHierarchy({viewHierarchy}: ViewHierarchyProps) {
+  const [selectedWindow] = useState(0);
+  return (
+    <Fragment>
+      <RenderingSystem system={viewHierarchy.rendering_system} />
```

Comment:
Thanks for pointing out that it looks like tabs, I'll need to follow up on the design for this because I wasn't sure how to render a single key-value pair nicely. I made this issue to track it #42997